tests/ten/ten_cor_014/run.py

```diff
@@ -18,10 +18,26 @@ def execute(self):
         tx_receipt1 = network.transact(self, web3_usr, storage.contract.functions.store(1), account_usr, storage.GAS_LIMIT)
         tx_receipt2 = network.transact(self, web3_usr, storage.contract.functions.store(2), account_usr, storage.GAS_LIMIT)
 
+        # create another user and have them send funds to our test user
+        pk2 = self.get_ephemeral_pk()
+        web3_usr2, account_usr2 = network.connect(self, private_key=pk2, check_funds=False)
```

Comment:
an ephemeral user without having it's funds checked wont be given funds so won't be able to transact, hence check_funds should be true